packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/EarlyAccessDropdown/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { Translation } from '@suite-components';
```

Comment:
<img width="778" alt="Screenshot 2021-11-23 at 10 19 54" src="https://user-images.githubusercontent.com/33235762/142998553-335b78f5-fc47-4885-a751-87de4b6006a0.png">

Reply:
oh, I haven't tried the hover, thanks for the catch